tests/src/analysis/testqgsprocessing.cpp

```diff
@@ -8998,6 +9000,135 @@ void TestQgsProcessing::parameterDxfLayers()
   QCOMPARE( dxfList.at( 0 ).layerOutputAttributeIndex(), dxfLayer.layerOutputAttributeIndex() );
 }
 
+void TestQgsProcessing::parameterPointCloudLayer()
```

Comment:
See below, but this should be ifdefed only if WITH_EPT

Reply:
Done.